pages/settings/profile.vue

```diff
@@ -12,14 +12,46 @@
 		tags: selfUserInfoStore.tags?.map(tag => tag.labelName),
 	}));
 
+	const userAvatarFileInput = ref<HTMLElement>();
+	/**
+	 * 点击头像事件，模拟点击文件上传并唤起文件资源管理器
+	 */
+	function handleUploadAvatarImage() {
+		userAvatarFileInput.value?.click();
+	}
+	
+	const userUploadFile = ref<string>();
 	const avatarCropperIsOpen = ref(false);
-	const handleOpenAvatarCropper = () => {
-		avatarCropperIsOpen.value = true;
-	};
+	/**
+	 * 开启图片裁切器事件（用户选择本地文件并上传事件）
+	 * @param e 应为用户上传文件的 input 元素的 change 事件
+	 */
+	function handleOpenAvatarCropper(e: Event) {
+		const fileInput = e?.target as HTMLInputElement;
```

Comment:
```suggestion
		const fileInput = e.target as HTMLInputElement;
```

根据类型声明，形参 `e` 一定不为空，不必添加 `?`。

Reply:
考虑到 HTML 元素和事件有可能因为各种原因为空，所以修改了函数的参数类型
多一层保险感觉安心一些
``` js
function handleOpenAvatarCropper(e?: Event) {
	const fileInput = e?.target as HTMLInputElement | undefined;
		if (fileInput?.files?.[0]) {
		...
		}
}
```